src/main/java/Duke/Handler.java

```diff
@@ -0,0 +1,79 @@
+package Duke;
+
+/**
+ * Class to handle commands given to Duke.
+ * @author Bryan Juniano
+ */
+
+public class Handler {
+    private enum Commands {
+        BYE, LIST, MARK, UNMARK, DELETE, TODO, EVENT, DEADLINE, FIND
+    }
+    public static Enum getCommand(String input){ //get the command
+        return Commands.valueOf(input.split(" ")[0].toUpperCase());
+    }
+
+    public static String[] getParameters(String input){ //get all tokens after the command
+        return input.split(" ", 2)[1].split(" ");
+    }
+
+    public static String[] parseParameters(Enum command, String[] parameters){ //get only the useful tokens
+        if(command.equals(Commands.MARK) || command.equals(Commands.UNMARK) || command.equals(Commands.DELETE) ||
+                command.equals(Commands.FIND)){
+            String[] cleanedParameters = new String[1];
+            cleanedParameters[0] = parameters[0];
+            return cleanedParameters;
+        }
+        else if(command.equals(Commands.TODO)){
```

Comment:
[There should be spaces between `if` and `(`, and `)` and `{`](https://se-education.org/guides/conventions/java/intermediate.html#loops:~:text=The%20if%2Delse%20class%20of%20statements%20should%20have%20the%20following%20form%3A).